source/Ship.cpp

```diff
@@ -2891,6 +2891,13 @@ bool Ship::IsReadyToJump(bool waitingIsReady) const
 	bool isJump = (jumpUsed.first == JumpType::JumpDrive);
 	double scramThreshold = attributes.Get("scram drive");
 
+	double departure = isJump ?
```

Comment:
~~last change I swear~~ can you add a comment here, maybe something like "If the system has a departure distance, the ship can only enter hyperspace if it is beyond that distance from the system center."

Reply:
<b>if it is the last one</b>
done